src/render.ts

```diff
@@ -164,408 +185,288 @@ export class ImageRenderer {
         }
-        const repr = provider.factory(this.reprCtx, provider.getParams)
+
+        const repr = provider.factory(this.reprCtx, provider.getParams);
         repr.setTheme({
-            color: this.reprCtx.colorThemeRegistry.create(params.colorTheme, { structure }, { carbonByChainId: false }),
+            color: this.reprCtx.colorThemeRegistry.create(params.colorTheme, { structure }, { carbonColor: { name: 'element-symbol' } }),
             size: this.reprCtx.sizeThemeRegistry.create(params.sizeTheme, { structure })
-        })
-        await repr.createOrUpdate({ ...provider.defaultValues, quality: params.quality || 'auto', ignoreHydrogens: true }, structure).run()
-        this.canvas3d.add(repr)
+        });
+        await repr.createOrUpdate({ ...provider.defaultValues, quality: params.quality || 'auto', ignoreHydrogens: true }, structure).run();
+        this.canvas3d.add(repr);
     }
 
     async addCartoon(structure: Structure, params: Partial<ReprParams> = {}) {
         await this.addRepresentation(structure, CartoonRepresentationProvider, {
             colorTheme: getColorTheme(structure),
             sizeTheme: 'uniform',
             ...params
-        })
+        });
     }
 
     async addGaussianSurface(structure: Structure, params: Partial<ReprParams> = {}) {
         await this.addRepresentation(structure, GaussianSurfaceRepresentationProvider, {
             colorTheme: getColorTheme(structure),
             sizeTheme: 'uniform',
             ...params
-        })
+        });
     }
 
     async addMolecularSurface(structure: Structure, params: Partial<ReprParams> = {}) {
         await this.addRepresentation(structure, MolecularSurfaceRepresentationProvider, {
             colorTheme: getColorTheme(structure),
             sizeTheme: 'uniform',
             ...params
-        })
+        });
     }
 
     async addBallAndStick(structure: Structure, params: Partial<ReprParams> = {}) {
         await this.addRepresentation(structure, BallAndStickRepresentationProvider, {
             colorTheme: 'element-symbol',
             sizeTheme: 'physical',
             ...params
-        })
+        });
     }
 
     async addCarbohydrate(structure: Structure, params: Partial<ReprParams> = {}) {
         await this.addRepresentation(structure, CarbohydrateRepresentationProvider, {
             colorTheme: 'carbohydrate-symbol',
             sizeTheme: 'uniform',
             ...params
-        })
+        });
     }
 
     /**
      * Creates PNG with the current 3dcanvas data
      */
     async createImage(outPath: string, size: StructureSize) {
         const occlusion = size === StructureSize.Big ? { name: 'on' as const, params: {
-            kernelSize: 4,
-            bias: 0.5,
+            samples: 32,
             radius: 64,
-        } } : { name: 'off' as const, params: {} }
+            bias: 0.5,
```

Comment:
The meaning of those values has changed, the current defaults are

```
{
  samples: 32,
  radius: 5,
  bias: 0.8,
  blurKernelSize: 15,
  resolutionScale: 1
}
```

Reply:
Updated.